### PR TITLE
Add simple example for verification of driver

### PR DESCRIPTION
**Pull request by saad-ali**
Follow up to https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/62

### CODE REVIEW

File: examples/demo.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+  storageClassName: csi-gce-pd
+
+---
+
+apiVersion: v1
+kind: ReplicationController
```
Comment (msau42) [inline examples/demo.yaml]: We recommend deployments or Statefulsets now?

File: examples/demo.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+kind: PersistentVolumeClaim
+metadata:
+  name: mypvc
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+  storageClassName: csi-gce-pd
```
Comment (msau42) [inline examples/demo.yaml]: Should the storageclass also be an example?

**Comment by davidz627** [comment]
There's already an example pod/pvc in deployment/kubernetes. Maybe it would make more sense to move it into an examples/ folder somewhere.

**Comment by davidz627** [comment]
Closing due to existing example

**Pull request closed without merging**